tools/kmod/sof_insert.sh

```diff
@@ -83,9 +83,11 @@ insert_module snd_soc_es8316
 insert_module snd_soc_es8326
 
 insert_module snd_soc_wm8960
+insert_module snd_soc_simple_card
 
 # core SOF driver
 insert_module snd_sof
+insert_module snd_sof_utils
```

Comment:
This should be at least a separate commit. Maybe even a separate PR if there's any debate about `snd_soc_simple_card`?

Reply:
unless simple_card depends on utils ? @AnneOnciulescu pls confirm.

Reply:
The commit message says "Also, snd_sof_utils was missing from sof_insert.sh." which is a pretty clear sign this is a separate bug fix.